plugins/codescene/api-report.md

```diff
@@ -0,0 +1,33 @@
+## API Report File for "@backstage/plugin-codescene"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="react" />
+
+import { BackstagePlugin } from '@backstage/core-plugin-api';
+import { IconComponent } from '@backstage/core-plugin-api';
+import { RouteRef } from '@backstage/core-plugin-api';
+
+// @public (undocumented)
+export const CodeSceneIcon: IconComponent;
+
+// @public (undocumented)
+export const CodescenePage: () => JSX.Element;
```

Comment:
Note that you have inconsistent casing of `CodeScene`, `Codescene`, and `codescene` in here. Not a huge deal, but might be useful to fix while we are at it. So uppercase things should be `CodeScene`, lowercase things should be `codeScene`